server/src/main/java/io/spine/system/server/SystemFeatures.java

```diff
@@ -44,10 +46,9 @@ interface SystemFeatures {
      * @return {@code true} if system events should be stored, {@code false} otherwise
      */
     boolean includePersistentEvents();
+
     /**
-     * Checks if the system events are allowed to be posted in parallel.
-     *
-     * @return {@code true} if it's OK to post system event is parallel, {@code false} otherwise
+     * Obtains an {@code Executor} with which system events will be posted.
      */
-    boolean postEventsInParallel();
+    Executor eventPostingExecutor();
```

Comment:
Exposing an `Executor` here means to show the implementation details. It's nobody's business on the level this high which executor is used. Besides, it is also dangerous, as anyone with an access to the system features may post anything at any time.

Please rethink this change so that the executor itself remains hidden.